pkg/machine/update.go

```diff
@@ -27,3 +27,13 @@ func UpdatePodmanDockerSockService(mc *vmconfigs.MachineConfig) error {
 
 	return nil
 }
+
+func ActivateRosettaService(mc *vmconfigs.MachineConfig) error {
+    args := []string{"sudo", "systemctl", "start", "rosetta-activation.service"}
+    if err := CommonSSH(mc.SSH.RemoteUsername, mc.SSH.IdentityPath, mc.Name, mc.SSH.Port, args); err != nil {
+        logrus.Warnf("Could not start rosetta-activation.service")
+        return err
+    }
+
+    return nil
```

Comment:
why do we need to activate this? can't we just have the unit enabled by default.

Reply:
so far this is defined in the ignition.go.
```
		rosettaUnit := Unit{
			Enabled:  BoolToPtr(false),
			Name:     "rosetta-activation.service",
```
If `Enabled:true` and `--rosetta=false` are set, the service will run automatically and an error will occur after the OS is booted. 
```
Feb 17 21:17:22 localhost.localdomain systemd[1]: Starting rosetta-activation.service - Activates Rosetta if necessary...
Feb 17 21:17:22 localhost.localdomain mount[1889]: mount: /var/mnt: wrong fs type, bad option, bad superblock on rosetta, missing codepage or helper program, or other error.
Feb 17 21:17:22 localhost.localdomain mount[1889]:        dmesg(1) may have more information after failed mount system call.
Feb 17 21:17:22 localhost.localdomain systemd[1]: rosetta-activation.service: Control process exited, code=exited, status=32/n/a
Feb 17 21:17:22 localhost.localdomain systemd[1]: rosetta-activation.service: Failed with result 'exit-code'.
Feb 17 21:17:22 localhost.localdomain systemd[1]: Failed to start rosetta-activation.service - Activates Rosetta if necessary.
```
There is no way to stop this. Because the machine VM can only be operated after the OS is booted.
Therefore, I think it is better to default to `Enabled:false` and start the service only when `--rosetta` is true.

Reply:
Well you can enable the unit in ignition only when rosetta was set to true.

Reply:
@Luap99  I explained in my latest comment.
https://github.com/containers/podman/pull/21670#issuecomment-1975002363